### PR TITLE
MT3-423 #ready-to-test Add disability page

### PR DESCRIPTION
**Pull request by LBHELewis**
# What?

Adds disability step

# Notes

Outstanding work: getting a dynamically populated list of tenants/household members. Displaying questions based on the answer to a previous question (so when you select that a tenant/household member has a disability it asks you about the nature of tenant X's disability https://balsamiq.cloud/sei56ua/pe10jok/r51CD).

### CODE REVIEW

File: __tests__/features/end-to-end.steps.ts [hunk 1]
```diff
@@ -557,6 +559,19 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
...
       await browser!.submit();
 
+      // Disability page
+      await expect(browser!.getCurrentUrl()).resolves.toContain(
+        "/disability?process_type=thc"
+      );
+      await browser!.findElement({ id: "disability-yes" }).click();
+      await browser!.findElement({ id: "who-disability-tenant-1" }).click();
+      await browser!.findElement({ id: "pip-or-dla-yes" }).click();
+      await browser!.findElement({ id: "who-pip-tenant-1" }).click();
```
Comment (CJWaszczuk) [inline __tests__/features/end-to-end.steps.ts]: Think this should be ` browser!.waitForEnabledElement({ id: id: "disability-yes" }).click();` etc

File: __tests__/features/end-to-end.steps.ts [hunk 2]
```diff
@@ -557,6 +559,19 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
...
+      await browser!
+        .wait(until.elementLocated({ name: "disability-notes" }), 500)
+        .sendKeys("Disability notes");
```
Comment (CJWaszczuk) [inline __tests__/features/end-to-end.steps.ts]: Could just be like `await browser!
        .waitForEnabledElement({ name: "door-mats-notes" })
        .sendKeys("Door mats notes");`
I'm assuming (touch wood) that `waitForEnabledElement` does the wait inside it.

**Review by CJWaszczuk — COMMENTED** [review]
Just a few bits with the feature test, I think the rest may be fine.

**Review by CJWaszczuk — APPROVED** [review]

File: __tests__/features/end-to-end.steps.ts (hunk 2 above)
Comment (erbridge) [reply __tests__/features/end-to-end.steps.ts]: It does :).

**Pull request merged**